glusterfs/server/setup.sls

```diff
@@ -58,14 +58,18 @@ glusterfs_vol_{{ name }}:
 {%- endif %}
 
 glusterfs_vol_{{ name }}_start:
+  {%- if force_compatibility %}
+  cmd.run:
+    - name: gluster volume start {{ name }}
+    - unless: gluster volume info {{ name }} | grep "Status: Started"
+    - require:
+      - cmd: glusterfs_vol_{{ name }}
+  {%- else %}
   glusterfs.started:
     - name: {{ name }}
     - require:
-      {%- if force_compatibility %}
```

Comment:
Why are you removing this? Compatibility mode also uses cmd to create volume so dependency needs to refer cmd and not glusterfs resource which is used in newer salt.